CHANGELOG

```diff
@@ -84,6 +84,14 @@ v3.9.0 (XXXX-XX-XX)
 
 * Upgrade bundled version of RocksDB to 6.26.
 
+* backport for PR 14916: The "Rebalance Shards" button now is displayed in a new tab, and it 
```

Comment:
Can you ensure CHANGELOG entries have line breaks after 80 columns at most? Additionally, sentences should start with a capital letter (e.g. `Backport` instead of `backport`).

Reply:
It's also better to use APM, BTS, etc. reference rather then a PR.